openstack/profiles/prodstack6

```diff
@@ -24,7 +24,7 @@ export GATEWAY=`openstack subnet show $EXT_SUBNET -c gateway_ip -f value`
 [[ -z "$FIP_RANGE" ]] && export FIP_RANGE="${FIP_RANGE_FIRSTIP}:${FIP_RANGE_LASTIP}"
 [[ -z "$CIDR_PRIV" ]] && export CIDR_PRIV="192.168.21.0/24"
 
-export UNDERCLOUD_OAM_NET="$(sed -E --quiet "s/.+OS_PROJECT_NAME=(.+)_project/net_\1-psd/p" ~/novarc)"
-export UNDERCLOUD_EXT_NET="$(sed -E --quiet "s/.+OS_PROJECT_NAME=(.+)_project/net_\1-psd-extra/p" ~/novarc)"
+export UNDERCLOUD_OAM_NET="$(echo net_${OS_USERNAME}-psd)"
+export UNDERCLOUD_EXT_NET="$(echo net_${OS_USERNAME}-psd-extra)"
```

Comment:
I don't think you need the echo here, can just use the variable directly and remove the ()

"${OS_USERNAME}-psd"
"${OS_USERNAME}-psd-extra"

Reply:
ow, you're right we could just expand them